stumpy/aampi.py

```diff
@@ -104,7 +119,21 @@ def __init__(self, T, m, egress=True, p=2.0, k=1):
         self._p = p
         self._k = k
 
-        mp = aamp(self._T, self._m, p=self._p, k=self._k)
+        if mp is None:
+            mp = aamp(self._T, self._m, p=self._p, k=self._k)
+        else:
+            mp = mp.copy()
+
+        if mp.shape != (
+            len(self._T) - self._m + 1,
+            2 * self._k + 2,
+        ):  # pragma: no cover
+            msg = (
+                f"The shape of `mp` must match ({len(T)-m+1}, {2 * k + 2})"
```

Comment:
Can you please add " but " (note the space after "but")

Reply:
```
f"The shape of `mp` must match ({len(T)-m+1}, {2 * k + 2}) but "
```

Otherwise, there is no space between this line and the next line

Reply:
I forgot adding dot or space. Thanks for pointing this out! This should be fixed now.